packages/api/src/record.ts

```diff
@@ -66,6 +69,20 @@ export type RecordUpdateOptions = {
  *
  * @beta
  */
+
+function removeUndefinedProperties(obj: any): any {
+  if (typeof obj !== 'object' || obj === null) return;
+  Object.keys(obj).forEach(key => {
+    const val = obj[key];
+    if (val === undefined) {
+      delete obj[key];
+    } else if (typeof val === 'object') {
+      removeUndefinedProperties(val);
+    }
+  });
+  return obj;
+}
+
```

Comment:
To avoid duplication use the function that [exists](https://github.com/TBD54566975/web5-js/blob/2bd93a85754b6f39208968ce2fc464bf943f15c2/packages/common/src/object.ts#L35) in `@web5/common`.

`import { removeUndefinedProperties } from '@web5/common';`

Reply:
Many tests fail when I switch to using the web5/common packages version.

Reply:
I am guessing it is because it assumes it will be a typed object of the one specified in the arguments of that version?